Automattic/jetpack
# Starter Plugin: Configure Sync to only sync minimum required data

#### Changes proposed in this Pull Request:

* In our starter plugin, configure sync to the minimal amount of data that needs to be synced

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Enable only Jetpack Starter Plugin
* Enable Jetpack Debug helper plugin
* Connect
* Go to Jetpack Debug > Sync Data Settings Tester
* Confirm you see a small list of thing and 4 items under jetpack_sync_callable_whitelist
